helm/nats-streaming-server/templates/deployment.yaml

```diff
@@ -64,6 +77,8 @@ spec:
         # Management port is enabled at default 8222
       - command:
           - /nats-streaming-server
+          - -sc
+          - "/etc/stan-config/stan.conf"
```

Comment:
All this nats-streaming stuff needs to be removed

Reply:
nats-streaming-server helm chart  should be removed